src/Campaigns/Actions/LoadCampaignDetailsAssets.php

```diff
@@ -33,12 +34,13 @@ public function __invoke()
         );
 
         $defaultForm = $campaign->defaultForm();
+        $defaultFormTitle = Utils::isV3Form($defaultForm->id) ? $defaultForm->settings->formTitle : $defaultForm->title;
         wp_localize_script($handleName, 'GiveCampaignDetails',
             [
                 'adminUrl' => admin_url(),
                 'currency' => give_get_currency(),
                 'isRecurringEnabled' => defined('GIVE_RECURRING_VERSION') ? GIVE_RECURRING_VERSION : null,
-                'defaultForm' => $defaultForm ? $defaultForm->settings->formTitle : null,
+                'defaultForm' => $defaultFormTitle,
```

Comment:
I don't see this being used anywhere. Also, this type of information can change without page reloading, so if you need this type of information it will be better to get it from the frontend.

Reply:
I had added this as part of the Default Form Widget work, but it is work-in-progress (and should probably be replaced with better state management).

Reply:
@alaca As @kjohnson mentioned, this is out of the scope of this PR, I only changed this line to make it work alongside the changes implemented on the `Campaign::defaultForm()` method as it now can retrieve both `V2` and `V3` forms.